Components/SaveLoad/AimePatches.cs

```diff
@@ -41,7 +41,7 @@ void Start()
 
         void Update()
         {
-            if (UnityEngine.Input.GetKey(KeyCode.F2))
+            if (UnityEngine.Input.GetKey(SettingsManager.instance.settings.AimeButton))
             {
                 shouldCoreBeReady = true;
             }
```

Comment:
I don't know if this is actually the intended behavior, but this would result in Aime card being read all the time after pressing AimeButton as shouldCoreBeReady value will always be true after pressing AimeButton
```suggestion
            shouldCoreBeReady = UnityEngine.Input.GetKey(SettingsManager.instance.settings.AimeButton);
```
This would result in having to press AimeButton every time you want to load your data though.

Reply:
Having it ready doesn't impact gameplay. I can only think of it being necessary to be triggerable if we decide to add multiple aimeID handling.